packages/page-alliance/package.json

```diff
@@ -1,16 +1,16 @@
 {
-  "bugs": "https://github.com/polkadot-js/apps/issues",
+  "bugs": "https://github.com/Cardinal-Cryptography/azero-dev/issues",
```

Comment:
I'm wondering if these won't cause problems during sync. I think they shouldn't, but I'm not sure 🤔

Reply:
It's an additional change in the diff, but since those changes are generated on every build anyway I'd keep them

Reply:
removed